Gemfile

```diff
@@ -6,7 +6,7 @@ gemspec
 
 gem 'activesupport', '>= 5.2.4.3'
 gem 'appraisal'
-gem 'bundler',       '~> 1.17'
+gem 'bundler',       '~> 2.2'
```

Comment:
This can be dropped from the gemfile. After bundler 1.17, bundler is smart enough to use the BUNDLED_WITH in the gemfile lock to switch bundler versions when necessary.